### DISCUSSION TITLE
Profile Page

### DISCUSSION THREAD

**Discussion by Bashamega** [cytrinc/bark-coding#148] [Q&A] [answered]
Hello:)
I know that it is quite early for the integration of the database. But I would like to start developing the account and profile pages.
I am going to create another branch called profiles, which I will add the version of the bark when the user is logged in and has a profile page.
Can I start working on it?

**Comment by Rubiidev-18** [comment, marked as answer]
yea, sure. but make it localstorage for now.

**Reply by Rubiidev-18** [reply]
I also made the branch for you!

**Reply by Bashamega** [reply]
Thanks

**Comment by Bashamega** [comment]
@Rubiidev-18 Can I add jest for testing some functionalities?

**Reply by Rubiidev-18** [reply]
alright!

**Comment by Bashamega** [comment]
I am creating thee register page. Should there be a minimum age?

**Reply by raynepaws** [reply]
> No, Bark should be accessible to anyone.

but we allow swear words

**Reply by Bashamega** [reply]
> It's super easy for one to lie about their age nowadays.

Correct

**Comment by Bashamega** [comment]
Hello @Rubiidev-18 :)
I have finished the register, and login page. I need a db to store the info in. currently it is an array consisting of objects
